hack/make.sh

```diff
@@ -44,6 +44,7 @@ echo
 DEFAULT_BUNDLES=(
 	validate-dco
 	validate-gofmt
+    validate-vet
```

Comment:
Not sure how that happened, I think my vim just did something weird, but its fixed now.